src/test/java/edu/iis/powp/gui/TestPlotSoftPatterns.java

```diff
@@ -43,9 +47,12 @@ private static void setupDrivers(Context context) {
 		context.addDriver("Client Plotter", clientPlotter);
 		Application.getComponent(DriverManager.class).setCurrentPlotter(clientPlotter);
 		
-		IPlotter plotter = new MyAdapter();
+		IPlotter plotter = new IPlotterDrawPanelControllerAdapter();
 		context.addDriver("Buggy Simulator", plotter);
 
+		IPlotter plotterWithSpecialLine = new LinePlotterAdapter();
```

Comment:
Cannot find LinePlotterAdapter. Make sure you add files to the repo before making a commit.